packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -109,6 +109,7 @@ export class Linker extends Component<LinkerProps, LinkerState> {
     this.handleExited = this.handleExited.bind(this);
     this.handleLinkSelected = this.handleLinkSelected.bind(this);
     this.isColumnSelectionValid = this.isColumnSelectionValid.bind(this);
+    this.reset = this.reset.bind(this);
```

Comment:
You don't need to do this, since you're not passing the function as a reference anywhere.